### PR TITLE
update world_bank scraper

### PR DESCRIPTION
**Pull request by cjyetman**
Caution: causes many changes/moves in the data CSV

### CODE REVIEW

**Comment by etiennebacher** [comment]
Hi, jumping in here to point out that most of the changes are due to the fact that the dataset is now sorted by `wb` and not by `country` anymore. Isn't sorting by `country` again an easy way to reduce the number of changes here? Sorry if I missed sth obvious about this

**Comment by cjyetman** [comment]
> Hi, jumping in here to point out that most of the changes are due to the fact that the dataset is now sorted by `wb` and not by `country` anymore. Isn't sorting by `country` again an easy way to reduce the number of changes here? Sorry if I missed sth obvious about this

Yes, but my goal here was to update the scraper with minimal changes, not to update the data (which is typically done all together in a separate process). Just pointing it out for @vincentarelbundock's awareness.

**Comment by NilsEnevoldsen** [comment]
It does make it hard to see what, if anything, changed as a result.

**Comment by vincentarelbundock** [comment]
Thanks all. I added one line of code with an `arrange` to make the diff easier to see. The changes look pretty minimal.

You can merge whenever you want.

**Review by vincentarelbundock — APPROVED** [review]

File: dictionary/data_world_bank.csv [hunk 1]
```diff
@@ -46,12 +46,12 @@ Comoros,COM
 "Congo, Dem. Rep.",COD
 "Congo, Rep.",COG
 Costa Rica,CRI
-Côte d'Ivoire,CIV
 Croatia,HRV
 Cuba,CUB
 Curaçao,CUW
 Cyprus,CYP
 Czech Republic,CZE
+Côte d’Ivoire,CIV
```
Comment (vincentarelbundock) [inline dictionary/data_world_bank.csv:54]: Do we want curly ' ?

Comment (cjyetman) [reply dictionary/data_world_bank.csv:54]: I think it is probably not proper, right? But that's what it is in the original data. Should I make the scraper "fix" it?

Comment (vincentarelbundock) [reply dictionary/data_world_bank.csv:54]: Yeah, I think so. My gut feeling is that converting to nice curly should be the typesetter's job, not a data-level thing.

**Comment by cjyetman** [comment]
@vincentarelbundock I was under the impression that all of these "getter" scripts got run together in a Docker during some part of the process you do, so I included the changed CSV just to facilitate reviewing the consequences of the changes I made, but with the intention of removing the changed CSV before merging. Looking a bit deeper at things now, it looks like maybe that doesn't happen anymore or the process has changed? Should I included the modified CSV as well, if it changes, when making a change to a getter script?

File: dictionary/data_world_bank.csv (hunk 1 above)
Comment (cjyetman) [reply dictionary/data_world_bank.csv:54]: 🤔 
``` r
packageVersion("countrycode")
#> [1] '1.5.0'
countrycode::countrycode("CIV", "iso3c", "country.name")
#> [1] "Côte d’Ivoire"
```

**Comment by vincentarelbundock** [comment]
> @vincentarelbundock I was under the impression that all of these "getter" scripts got run together in a Docker during some part of the process you do, so I included the changed CSV just to facilitate reviewing the consequences of the changes I made, but with the intention of removing the changed CSV before merging. Looking a bit deeper at things now, it looks like maybe that doesn't happen anymore or the process has changed? Should I included the modified CSV as well, if it changes, when making a change to a getter script?

Yeah, it's been a while, but if I remember correctly, my previous setup was ridiculously over-(and badly-) engineered. So I simplified everything. `get_*()` saves a CSV file that we keep in the repo. Then, `build.R` reads and merges all the CSV files. 

I always run it on my local machine; never on Docker. Maybe I should do that, but things have worked ok thus far...

File: dictionary/data_world_bank.csv (hunk 1 above)
Comment (cjyetman) [reply dictionary/data_world_bank.csv:54]: It's hard to see the difference with GitHub's formatting, but both ASCII single quote and UTF curly quote work, and UTF curly quote is what's currently in `countrycode::codelist`, so I'm assuming leaving it as curly quote is ok, maybe even ideal.
``` r
library(countrycode)
packageVersion("countrycode")
#> [1] '1.6.0'

as.hexmode(utf8ToInt("'"))
#> [1] "27"
as.hexmode(utf8ToInt("’"))
#> [1] "2019"

countrycode("Côte d\U27Ivoire", "country.name", "country.name")
#> [1] "Côte d’Ivoire"
countrycode("Côte d\U2019Ivoire", "country.name", "country.name")
#> [1] "Côte d’Ivoire"

stringi::stri_escape_unicode(countrycode("CI", "iso2c", "country.name"))
#> [1] "C\\u00f4te d\\u2019Ivoire"
```

**Pull request merged**